frontend/src/container/ServiceTable/Filter/FilterDropdown.tsx

```diff
@@ -0,0 +1,41 @@
+import { SearchOutlined } from '@ant-design/icons';
+import { Button, Card, Input, Space } from 'antd';
+import type { FilterDropdownProps } from 'antd/es/table/interface';
+
+import { SEARCH_PLACEHOLDER } from '../Columns/ColumnContants';
+
+export const filterDropdown = ({
+	setSelectedKeys,
+	selectedKeys,
+	confirm,
+}: FilterDropdownProps): JSX.Element => {
+	const handleSearch = (): void => {
```

Comment:
this is the redundant function please remove this one

Reply:
This should be as it is, as the function `confirm` is expecting the optional field which is `param?: FilterConfirmProps | undefined` which is not throwing an error `(param?: FilterConfirmProps | undefined) => void' is not assignable to type 'KeyboardEventHandler<HTMLInputElement>'`